### PR TITLE
Dependency Notation for Horologist.

### LINKED ISSUE #563: Dependency notations for Horologist
## About the library(-ies)

- *Description*: Horologist is a group of libraries that aim to supplement Wear OS developers with features that are commonly required by developers but not yet available. 
- *GitHub*: https://github.com/google/horologist
- *Official website* (if any, besides the GitHub page): https://google.github.io/horologist/

## Want to contribute the sample?

Yep.

### PR DESCRIPTION
**Pull request by yschimke**
## What?

Resolves #563

Adds dependency notation for https://github.com/google/horologist

## Why?

Trying out refreshVersions for my Wear app development, and these libraries are likely to be required for most properly implemented wear apps.

## How?

Followed instructions page https://jmfayard.github.io/refreshVersions/contributing/submitting-prs/dependency-notations-updates/ and the approach used by Accompanist.

## Testing?

Ran gradle test command.

### CODE REVIEW

File: plugins/dependencies/src/test/resources/dependencies-versions-key-validated.txt [hunk 1]
```diff
@@ -388,6 +388,15 @@ com.google.android.gms..play-services-vision=version.google.android.play-service
...
 com.google.android.gms..strict-version-matcher-plugin=version.NO-RULE
+com.google.android.horologist..horologist-audio-ui=version.NO-RULE
+com.google.android.horologist..horologist-audio=version.NO-RULE
+com.google.android.horologist..horologist-composables=version.NO-RULE
+com.google.android.horologist..horologist-media-data=version.NO-RULE
+com.google.android.horologist..horologist-media-ui=version.NO-RULE
+com.google.android.horologist..horologist-media3-backend=version.NO-RULE
+com.google.android.horologist..horologist-media=version.NO-RULE
+com.google.android.horologist..horologist-network-awareness=version.NO-RULE
+com.google.android.horologist..horologist-tiles=version.NO-RULE
```
Comment (jmfayard) [inline plugins/dependencies/src/test/resources/dependencies-versions-key-validated.txt:399]: see here for the version

File: plugins/dependencies/src/main/kotlin/dependencies/Google.kt [hunk 2]
```diff
@@ -595,6 +596,50 @@ object Google : IsNotADependency {
...
+    /**
+     * Horologist is a group of libraries that aim to supplement
+     * Wear OS developers with features that are commonly required
+     * by developers but not yet available.
+     *
+     * - [Official website here](https://google.github.io/horologist/)
+     * - GitHub page: https://github.com/google/horologist
+     */
+    @Incubating
+    object Horologist : DependencyGroup(group = "com.google.android.horologist") {
```
Comment (jmfayard) [inline plugins/dependencies/src/main/kotlin/dependencies/Google.kt:610]: Missing a rule, see file dependencies-versions-key-validated.txt

Shouuld be something like this:

```
        rawRules = """
            com.google.android.horologist:*
                ^^^^^^^        ^^^^^^^^^^
        """.trimIndent()
```

File: plugins/dependencies/src/main/kotlin/dependencies/Google.kt [hunk 3]
```diff
@@ -595,6 +596,50 @@ object Google : IsNotADependency {
...
+
+        val audio = Audio
+
+        object Audio : DependencyNotationAndGroup(group = "com.google.android.horologist", name = "horologist-audio") {
+            val ui = module("horologist-audio-ui")
+        }
+
+        val composables = module("horologist-composables")
```
Comment (jmfayard) [inline plugins/dependencies/src/main/kotlin/dependencies/Google.kt:618]: Can you put this module and the two others at line 638 and 640 first?

File: plugins/dependencies/src/main/kotlin/dependencies/Google.kt [hunk 4]
```diff
@@ -595,6 +596,50 @@ object Google : IsNotADependency {
...
+
+        object Compose : DependencyGroup(group = "com.google.android.horologist") {
```
Comment (jmfayard) [inline plugins/dependencies/src/main/kotlin/dependencies/Google.kt:620]: Missing `val compose = Compose` no?

**Review by jmfayard — CHANGES_REQUESTED** [review]
Thanks for your contribution! Left some comments

**Comment by jmfayard** [comment]
Merged, great job, thank you! 🙏🏻

**Review by jmfayard — APPROVED** [review]

**Comment by LouisCAD** [comment]
Nicely done, and right from the library authors, thank you! 🙌🙏

**Pull request merged**